tests/karate/src/test/java/fe/features/wallet.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Wallet
+
+  Scenario: Open the app for the first time and see the wallet seed
+    When call read('classpath:fe/utils/platform.feature') { name: 'open_app' }
```

Comment:
It is useful to define classpaths as a variable in a feature file for constants, to make it more readable and easier to change in the future (see `constants.feature` in be for an example)

Reply:
Done